app/services/album.js

```diff
@@ -6,10 +6,12 @@ const axios = require('axios'),
   logger = require('../logger'),
   url = config.common.albumsApi.url;
 
-exports.getAll = async source => {
-  const data = await axios.get(`${url}${source}`).catch(err => {
-    logger.error(err);
-    throw errors.albumsApiError(err);
-  });
-  return data.data;
+exports.getAlbums = source => {
+  return axios
```

Comment:
Remove brackets and return, also the method implementation is so generic and name isn't

Reply:
but, the method just do get albums, all or by id
what name do you suggest?